hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -917,6 +918,7 @@ public long getToVersion() {
     
     public static interface Validator {
         void validate(HollowProducer.ReadState readState);
+        void lock();
```

Comment:
With Hollow 3, now we can use a default method and avoid the breaking change.

Reply:
we've used freeze in the past?